### PR TITLE
Added Trivia to UNLaM second Scene.

### PR DESCRIPTION
**Pull request by santiagons**

### CODE REVIEW

File: Assets/Scripts/TriviaScript.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		btnError2=GameObject.Find("btnRespuestaIncorrecta2").GetComponent<Button>();
+		audioSourceOk=GameObject.Find("AudioCorrect").GetComponent<AudioSource>();
+		audioSourceFail=GameObject.Find("AudioFail").GetComponent<AudioSource>();
+	}
+
+	public void responderError1() {
+		btnError1.image.color = Color.red;
+		resaltarRespuestaCorrecta ();
+		reproducirError ();
+		anularBotones ();
```
Comment (fedemoglia) [inline Assets/Scripts/TriviaScript.cs]: The last 3 lines are repeated in the function below also. Maybe you can extract them to a new function? It's a desirable change, but don't worry in absolute about it.

Comment (santiagons) [inline Assets/Scripts/TriviaScript.cs]: They are also in spanish. So yeah, I'll fix it. Thanks!

Comment (santiagons) [inline Assets/Scripts/TriviaScript.cs]: There :)

Comment (fedemoglia) [inline Assets/Scripts/TriviaScript.cs]: Nice!

**Comment by nogueirasjorge** [comment]
@santiagons There are some conflicts on this pull request. You should solve those conflicts and update the pull request. Personally I can't help you because those changes are in unity files.

**Comment by santiagons** [comment]
Yes, we noticed it last night (2/10) with Federico. The conflicts emerged because HISTO-40 was merged to develop and it includes modifications in c-unlam-2.unity. 
I have to coordinate some work with Federico to solve this. Thanks!

**Pull request closed without merging**